### PR TITLE
Update documentation on nested / autonomous tasks

### PR DESCRIPTION
**Pull request by zzstoatzz**

### CODE REVIEW

File: docs/concepts/tasks.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Tasks are special because they receive metadata about upstream dependencies and
 
 Tasks also take advantage of automatic Prefect [logging](/concepts/logs/) to capture details about task runs such as runtime, tags, and final state.
 
-You can define your tasks within the same file as your flow definition, or you can define tasks within modules and import them for use in your flow definitions. All tasks must be called from within a flow. Tasks may not be called from other tasks.
+You can define your tasks within the same file as your flow definition, or you can define tasks within modules and import them for use in your flow definitions. Tasks may be called from other tasks from within a flow, from within a subflow, or (as of `prefect 2.18.x`) from within another task.
```
Comment (zzstoatzz) [inline docs/concepts/tasks.md:35]:
```suggestion
You can define your tasks within the same file as your flow definition, or you can define tasks within modules and import them for use in your flow definitions. Tasks may be called from other tasks, from within a flow, from within a subflow, or (as of `prefect 2.18.x`) from within another task.
```

File: docs/concepts/tasks.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ Tasks are special because they receive metadata about upstream dependencies and
 
 Tasks also take advantage of automatic Prefect [logging](/concepts/logs/) to capture details about task runs such as runtime, tags, and final state.
 
-You can define your tasks within the same file as your flow definition, or you can define tasks within modules and import them for use in your flow definitions. All tasks must be called from within a flow. Tasks may not be called from other tasks.
+You can define your tasks within the same file as your flow definition, or you can define tasks within modules and import them for use in your flow definitions. Tasks may be called from other tasks, from within a flow, from within a subflow, or (as of `prefect 2.18.x`) from within another task.
```
Comment (serinamarie) [inline docs/concepts/tasks.md:35]: What is the difference between "tasks may be called from other tasks" and "from within another task"?

File: docs/tutorial/tasks.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-description: Learn the basics of adding tasks to our flow.
+description: Learn the basics of writing tasks.
```
Comment (serinamarie) [inline docs/tutorial/tasks.md:2]: ❤️

File: docs/concepts/tasks.md (hunk 2 above)
Comment (zzstoatzz) [reply docs/concepts/tasks.md:35]: ummmm.... nothing. lemme update

Comment (zzstoatzz) [reply docs/concepts/tasks.md]: updated in [rm duplicative language](https://github.com/PrefectHQ/prefect/pull/13154/commits/b27dd0c0a5a7f5e2787eb6dea3b3301debc498d3)

File: docs/tutorial/tasks.md [hunk 4]
```diff
@@ -25,8 +25,8 @@ Flows and tasks share some common features:
 
 Network calls (such as our `GET` requests to the GitHub API) are particularly useful as tasks because they take advantage of task features such as [retries](/concepts/tasks/#retries), [caching](/concepts/tasks/#caching), and [concurrency](/concepts/task-runners/#using-a-task-runner).
 
-!!! warning "Tasks must be called from flows"
-    All tasks must be called from within a flow. Tasks may not call other tasks directly.
+!!! warning "Tasks may be called from other tasks"
```
Comment (serinamarie) [inline docs/tutorial/tasks.md:28]: Is this still a `warning` admonition? Or is there a happier/less ominous choice (such as `tip` or `info`)

Comment (zzstoatzz) [inline docs/tutorial/tasks.md:28]:
```suggestion
!!! tip "Tasks may be called from other tasks"
```

Comment (zzstoatzz) [reply docs/tutorial/tasks.md]: good call, make it `tip` in [869bb7a](https://github.com/PrefectHQ/prefect/pull/13154/commits/869bb7aec7a7efc8768e26966d2e6157b32b3b02)

File: docs/concepts/tasks.md [hunk 5]
```diff
@@ -32,7 +32,7 @@ Tasks are special because they receive metadata about upstream dependencies and
 
 Tasks also take advantage of automatic Prefect [logging](/concepts/logs/) to capture details about task runs such as runtime, tags, and final state.
 
-You can define your tasks within the same file as your flow definition, or you can define tasks within modules and import them for use in your flow definitions. Tasks may be called from other tasks, from within a flow, from within a subflow, or (as of `prefect 2.18.x`) from within another task.
+You can define your tasks within the same file as your flow definition, or you can define tasks within modules and import them for use in your flow definitions. Tasks may be called from within a flow, from within a subflow, or (as of `prefect 2.18.x`) from within another task.
```
Comment (serinamarie) [inline docs/concepts/tasks.md:35]: I notice that below this paragraph, we have an example called "Calling a task from a flow" but none called "Calling a task within a task". Not blocking, but maybe an example would be nice.

**Review by serinamarie — APPROVED** [review]
Left one thought for maybe a future PR (or this one), but LGTM!

**Review by serinamarie — APPROVED** [review]

**Pull request merged**